### PR TITLE
Prepare release 0.100.0

### LINKED ISSUE #2929: Release the operator v0.100.0

### PR DESCRIPTION
**Pull request by VineethReddy02**
resolves #2929

### CODE REVIEW

**Comment by pavolloffay** [comment]
@VineethReddy02 could you please update the changelog. This PR was merged https://github.com/open-telemetry/opentelemetry-operator/pull/2938/files#diff-ad5afc4cd0359615bbf52013394a16ff12b136d4e62066e576dd0c0295b2ea22R1

File: CHANGELOG.md
Comment (pavolloffay) [inline CHANGELOG.md:34]: this should be 1.32.1

**Review by frzifus — APPROVED** [review]

File: CHANGELOG.md
Comment (pavolloffay) [reply CHANGELOG.md:53]: The version still hasn't been updated

File: RELEASE.md
Comment (TylerHelmuth) [reply RELEASE.md:47]: So close lol

**Comment by pavolloffay** [comment]
I would like to block this until https://github.com/open-telemetry/opentelemetry-operator/pull/2878 is merged

**Comment by pavolloffay** [comment]
Changelog for this can be added as well https://github.com/open-telemetry/opentelemetry-operator/pull/2964#pullrequestreview-2063141378

**Comment by VineethReddy02** [comment]
Okay, I see we are waiting for #2878 (I will update chlog once its merged)

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**
